mycroft/messagebus/send_func.py

```diff
@@ -32,10 +30,7 @@ def send(message_to_send, data_to_send=None):
     data_to_send = data_to_send or {}
 
     # Calculate the standard Mycroft messagebus websocket address
-    config = Configuration.get([DEFAULT_CONFIG,
-                                SYSTEM_CONFIG,
-                                USER_CONFIG],
-                               cache=False)
+    config = Configuration.get(cache=False, remote=False)
```

Comment:
confused as to why it matters if the remote config is accessed here.

Reply:
This just replicates the existing behavior instead of the list of locations it uses the remote=False flag. So I don't think investigating this belongs in this PR

Originally it was set up like this to speed up the command line for sending messages to mycroft core.

Reply:
Since this only replicate the earlier behavior using the flag instead of the location list, I think this can be kept as is.

Originally remote was dropped to get some speedup when using the command-line application to inject messages.

Reply:
is it really that much faster?  Regardless, we don't have to address it in this PR.  just asking a question.